okio/jvm/src/main/java/okio/platform.kt

```diff
@@ -17,6 +17,8 @@
 package okio
 
 actual typealias JvmOverloads = kotlin.jvm.JvmOverloads
+actual typealias JvmField = kotlin.jvm.JvmField
```

Comment:
doesn’t work as I was hoping:
https://youtrack.jetbrains.com/issue/KT-18882#comment=27-2847603

Reply:
Damn. Revert?

Reply:
Something. I don’t wanna release 2.0 until it’s fixed or reverted.

Reply:
It does work, it's the IDE plugin that is broken. I decompiled `Segment` with JD-GUI and I see that it has `public final` for fields that are marked with `@okio.JvmField` and no getters. If I create the following Java file:
```
public class Test {
  public static void main(String[] args) {
    Segment segment = new Segment();
    segment.getData();
  }
}
```
The IDE won't complain, but compiling it from command line will fail with `cannot find symbol: method getData()`. Changing this to `segment.data` works fine with both IDE and compiler.

Reply:
Whoa! Thats great news. Weird that the IDE plugin can disagree here.

I'll try to figure out what bug to report.